packages/styles/forms.css

```diff
@@ -58,7 +60,6 @@ textarea,
 [role='menuitemcheckbox'],
 [role='menuitemradio'],
 [role='textbox'],
-[aria-haspopup='listbox'],
```

Comment:
What were these styles for?

Reply:
These styles were overly aggressive and getting applied incorrectly. The prior styles were assuming that anything that had `[aria-haspopup='listbox']` should be styled like an input - which is incorrect.